tests/unit/scripting.tcl

```diff
@@ -591,27 +591,27 @@ start_server {tags {"scripting"}} {
         run_script {
             local i
             local x={}
-            redis.call('del','mylist')
+            redis.call('del',KEYS[1])
             for i=1,100 do
                 table.insert(x,i)
             end
-            redis.call('rpush','mylist',unpack(x))
-            return redis.call('lrange','mylist',0,-1)
+            redis.call('rpush',KEYS[1],unpack(x))
+            return redis.call('lrange',KEYS[1],0,-1)
         } 1 mylist
     } {1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100}
 
     test {Number conversion precision test (issue #1118)} {
         run_script {
               local value = 9007199254740991
-              redis.call("set","foo",value)
-              return redis.call("get","foo")
+              redis.call("set",KEYS[1],value)
+              return redis.call("get",KEYS[1])
         } 1 foo
     } {9007199254740991}
```

Comment:
i don't think we have to do that.
if this problem is in just one or two tests, we can align them with others.
but if the majority of the change here are similar, i'd rather drop them, and just add the missing command arguments

Reply:
fixed in https://github.com/redis/redis/pull/11134/commits/c4f30dede4260edfb10c79db6dda08180877bae2